utilities/cloudharness_utilities/deployment-configuration/helm/templates/auto-database-postgres-backup.yaml

```diff
@@ -38,9 +38,9 @@ spec:
                 - name: BACKUP_SUFFIX
                   value: {{ .root.Values.backup.suffix | quote }}
                 - name: BACKUP_DIR
-                  value: {{ .root.Values.backup.dir | quote }}
+                  value: {{ (printf "%s/postgres/%s" .root.Values.backup.dir .app.harness.database.name) | quote }}
```

Comment:
This could backfire since one can specify the same database name in two applications causing accidental overwrite of backups, without even noticing it until one needs that backup.

If backups should be stored on one volume across dbs, we need to ensure that one DB backup cannot conflict with another DB backup.

Reply:
usually applications shouldn't specify the database name as it is automatically assigned from app name. But we really need it to be configurable for backwards compatibility (otherwise we cannot use auto-database with application that use a db with different name than default)